recipes/libgcrypt/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package C)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+find_library(CAP_LIBRARY NAMES libcap.a libcap.so)
```

Comment:
This does not belong in a test package.
If libgcrypt needs libcap, then libcap should be added to cci as well.

Reply:
I cannot understand how handle this situation. On the one hand some compilers link libcap without specification them explicitly, maybe it is shipped with libc or another standard library, on the other hand there are compilers those require libcap to be explicitly specified in the parameters

Reply:
We have system packages which are avoided but that might be a solution if its uneven

Reply:
My Linux system does not have libcap.so/libcap.a. (it has /usr/lib64/libcap.so.2 though)

```
$ sudo dnf whatprovides /usr/lib64/libcap.so.2
libcap-2.48-2.fc34.x86_64 : Library for getting and setting POSIX.1e capabilities
Repo        : @System
Matched from:
Filename    : /usr/lib64/libcap.so.2
$ sudo dnf info libcap-2.48-2.fc34.x86_64
Installed Packages
Name         : libcap
Version      : 2.48
Release      : 2.fc34
Architecture : x86_64
Size         : 182 k
Source       : libcap-2.48-2.fc34.src.rpm
Repository   : @System
From repo    : anaconda
Summary      : Library for getting and setting POSIX.1e capabilities
URL          : https://sites.google.com/site/fullycapable/
License      : BSD or GPLv2
Description  : libcap is a library for getting and setting POSIX.1e (formerly POSIX 6)
             : draft 15 capabilities.
```

Looks like libcap is available here: https://git.kernel.org/pub/scm/libs/libcap/libcap.git/refs/

And lucky you, libcap is available already on cci...

Reply:
Is this library even needed at all?

Reply:
Looks like it depends on the configure option `--with-capabilityies`.

https://github.com/gpg/libgcrypt/blob/ccfa9f2c1427b40483984198c3df41f8057f69f8/configure.ac#L582-L587
->
https://github.com/gpg/libgcrypt/blob/ccfa9f2c1427b40483984198c3df41f8057f69f8/configure.ac#L2221-L2248

Reply:
Lucky you, not me. If you give a remark you are interested in this detail more than I

Reply:
Lucky in the sense that a libcap recipe should not be created.
I think you need to add the requirement to the main recipe.

Reply:
Agreed. I don't add extra requirement to test package. Any package should solve its dependencies without extra step.